client/middleware/auth.ts

```diff
@@ -10,6 +10,7 @@ export default defineNuxtRouteMiddleware((to, from) => {
     }
   } else if (!isAuthenticated.value) {
     if (ANONYMOUS_ROUTES.includes(from.path)) return abortNavigation()
-    return navigateTo('/login', { replace: true })
+    const redirectedFrom = to.path === '/' || to.path === '/files' ? '' : `?redirectedFrom=${encodeURI(to.path)}`
```

Comment:
Leave `/files`, we may change de landing page in the future... it is not a concern for it to appear in the URL.